models/tasks.js

```diff
@@ -112,10 +112,11 @@ const fetchTask = async (taskId) => {
  * @return {Promise<tasks|Array>}
  */
 
-const fetchUserTasks = async (username, statuses = []) => {
+const fetchUserTasks = async (username, statuses = [], hasUser = false) => {
   try {
     const { user } = await fetchUser({ username })
```

Comment:
A small suggestion @swarajpure and @sumitd94. Do we really need to fetch user object ? As we are using username entity of that object in whole function which we already have. We can directly call `userUtils.getUserId(username)` Let me  know if this is helpful/valid.

Reply:
Great question @bhk31!
We should ideally use the util (which does also make a read request to get the user, but it can be sped up in future by caching)